builder/core/data.py

```diff
@@ -338,6 +338,48 @@ class PKG_TOOLS(Enum):
             'osx_architectures': 'arm64;x86_64'
         },
     },
+    'tvos': {
+        'cmake_args': [
+            '-GXcode',
+            '-DCMAKE_SYSTEM_NAME=tvOS',
+            '-DCMAKE_OSX_ARCHITECTURES="{osx_architectures}"',
+            '-DCMAKE_OSX_DEPLOYMENT_TARGET={osx_deployment_target}'
+        ],
+        'run_tests': False,
+        'architectures': {
+            'armv8': {
+                'variables': {
+                    'osx_architectures': 'arm64'
+                }
+            }
+        },
+        'variables': {
+            'exe': '',
+            'osx_deployment_target': '13.0',
```

Comment:
dont think this is correct. the latest version of tvOS is 7.0 and we should proably target two before so 5.0. 13 is the iOS version I believe.

Reply:
good catch